sdk/identity/identity/Troubleshooting.md

```diff
@@ -0,0 +1,251 @@
+| [Azure Kubernetes Service](https://docs.microsoft.com/azure/aks/use-managed-identity)                                              | [Configuration Instructions](https://docs.microsoft.com/azure/aks/use-managed-identity)                                                                    |
+| [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/msi-authorization)                                                |                                                                                                                                                            |
+| [Azure Arc](https://docs.microsoft.com/azure/azure-arc/servers/managed-identity-authentication)                                    | [Configuration Instructions](https://docs.microsoft.com/azure/azure-arc/servers/security-overview#using-a-managed-identity-with-azure-arc-enabled-servers) |
+| [Azure Service Fabric](https://docs.microsoft.com/azure/service-fabric/concepts-managed-identity)                                  | [Configuration Instructions](https://docs.microsoft.com/azure/service-fabric/configure-existing-cluster-enable-managed-identity-token-service)             |
+
+## Troubleshooting Visual Studio Code Authentication Issues
+
+### Credential Unavailable
+
+#### Failed To Read VS Code Credentials / Authenticate via Azure Tools plugin in VS Code.
+
+The `VS Code Credential` failed to read the credential details from the cache.
+
+The Visual Studio Code authentication is handled by an integration with the Azure Account extension.
+To use this form of authentication, ensure that you have installed the Azure Account extension,
+then use View > Command Palette to execute the Azure: Sign In command. This command opens a browser window and displays a page that allows you
+to sign in to Azure. After you've completed the login process, you can close the browser as directed. Running your application
+(either in the debugger or anywhere on the development machine) will use the credential from your sign-in.
+
+If you already had the Azure Account extension installed and had logged in to your account. Then try logging out and logging in again, as
+that will re-populate the cache on the disk and potentially mitigate the error you're getting.
+
+If you have used the VSCode extension to authenticate once and if you are using the `DefaultAzureCredential` outside of the VSCode, it will try to authenticate with the VSCode credentials. This is important, because if you stop using VSCode for a while, your VSCode auth token will eventually get expired, and your DefaultAzureCredentials will begin to fail. In such cases, you have to log out on the VSCode extension (and log in again if you want to continue using it).
```

Comment:
I know you took this in verbatim from what I said, but I’m sure there are better ways to write this.